package-lock.json

```diff
@@ -0,0 +1,12440 @@
+{
+  "name": "formester",
```

Comment:
delete package-lock file and install with yarn

Reply:
Fixed